Runtime/Stats/Components/StatListener.cs

```diff
@@ -17,6 +17,7 @@ public class StatListener : MonoBehaviour
         private void Start()
         {
             m_WatchedStatHolder.statContainer.GetStat(m_WatchedStat).onValueChanged += m_Event.Invoke;
+            m_Event.Invoke(m_WatchedStatHolder.statContainer.GetStat(m_WatchedStat).statValue);
```

Comment:
Just a tiny nitpick: `m_WatchedStatHolder.statContainer.GetStat(m_WatchedStat)` can be cached into a variable to avoid finding the stat twice. Other than that, loooks goo to me. ;)